samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -14,8 +14,8 @@
 
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./lib/Api</directory>
-            <directory suffix=".php">./lib/Model</directory>
+            <directory suffix=".php">./lib\Api</directory>
```

Comment:
@serezha93 looks like you're using Windows. Does this phpunit.xml.dist file work for you in your Windows machine?

I think it should be updated to .\lib\Api instead.